marklogic-data-hub-central/ui/src/components/entities/matching/matching-step-detail/matching-step-detail.tsx

```diff
@@ -681,6 +670,31 @@ const MatchingStepDetail: React.FC = () => {
     }
   };
 
+  const handleAddMenu = (key) => {
+    if (key ==="singlePropertyRuleset") {
```

Comment:
I'm not sure why the `lint` task didn't catch these, but please add a space between `=== "`